README.md

```diff
@@ -43,33 +43,35 @@ Mapping the dependencies between scenarios is done by annotating the steps with
 you buy a new postcard
     [Documentation]    *model info*
     ...    :IN: None
-    ...    :OUT: new postcard | postcard.wish=None
+    ...    :OUT: new postcard
 
 you have a blank postcard
     [Documentation]    *model info*
     ...    :IN: postcard.wish==None
-    ...    :OUT: postcard.wish==None
+    ...    :OUT: postcard.wish=None
 
 you write '${your_wish}' on the postcard
     [Documentation]    *model info*
     ...    :IN: postcard.wish==None
     ...    :OUT: postcard.wish=${your_wish}
 ```
 
-The first scenario can be executed directly. It has no dependencies that need to be resolved before going into its first step, as indicated by the `:IN:` expression which is `None`. After completing the step, a new domain term is available with a single property. The term `postcard` with property `wish` is introduced, as stated by the `:OUT:` expressions. This satisfies the condition for the then-step to complete this scenario.
+The first scenario has no dependencies and can be executed at any time. This is evident from the fact that the scenario does not have any given-steps. From the `*model info*` we see that no dependencies need to be resolved before going into the first step: _When you buy a new postcard_. This is indicated by the `:IN:` expression which is `None`. After completing the step, a new domain term is available, `postcard`, as a result of the `:OUT:` statement `new postcard`. The following then-step adds detail to the postcard by adding a property. Setting `postcard.wish=None` in the `:OUT:` statement indicates that _you have a blank postcard_.
```

Comment:
In theory we treat it as being "TRUE".

Reply:
This is an interesting one. Up to now I read this as `None` meaning that there is no condition. But in all other cases we process the expression to a truth value. This seems contradictory, because (in Python) `None` evaluates to a truth value of `False`.

Reply:
I propose to pick this up as a separate item. If we choose to change this, it also implies a code and interface change.

Reply:
Without doing any functional changes under this PR, I at least changed de wording to prevent saying that `None` is an expression.